### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by trentonf96**

### CODE REVIEW

File: cst438_gradebook/src/main/java/com/cst438/services/RegistrationServiceMQ.java [hunk 1]
```diff
@@ -44,16 +44,23 @@ public RegistrationServiceMQ() {
...
 	}
 
 	// sender of messages to Registration Service
 	@Override
 	public void sendFinalGrades(int course_id, CourseDTOG courseDTO) {
 		 
 		//TODO  complete this method in homework 4
-		
+	   System.out.println("Sending grades for course " + course_id + " to register MQ service");
+	   rabbitTemplate.convertAndSend(registrationQueue.getName(), courseDTO);
```
Comment (BowieRules) [inline cst438_gradebook/src/main/java/com/cst438/services/RegistrationServiceMQ.java:62]: Should we set course_id for the courseDTO object here? I don't think it knows which course to update grades for...

**Review by BowieRules — COMMENTED** [review]
That's the only piece that is missing, I think?

File: cst438_gradebook/src/main/java/com/cst438/services/RegistrationServiceMQ.java (hunk 1 above)
Comment (trentonf96) [reply cst438_gradebook/src/main/java/com/cst438/services/RegistrationServiceMQ.java:62]: I believe the course_id is set in the API
githubweek4

**Comment by BowieRules** [comment]
You are correct, not sure why the method is accepting a course_id
separately from CourseDTOG object... Mhm?!?!

public void sendFinalGrades(int course_id, CourseDTOG courseDTO)

**Comment by trentonf96** [comment]
I think RegistrationServiceREST.java makes use of the course id

File: cst438_gradebook/src/main/java/com/cst438/services/RegistrationServiceMQ.java (hunk 1 above)
Comment (trentonf96) [reply cst438_gradebook/src/main/java/com/cst438/services/RegistrationServiceMQ.java:62]: I think RegistrationServiceREST.java makes use of the course id

**Review by BowieRules — APPROVED** [review]
@trentonf96 since we tested it on our machines and the code looks clean, I approve. Please go ahead with the merge.

**Pull request merged**